sample-apps/react/react-dogfood/package.json

```diff
@@ -35,7 +35,7 @@
     "react-form": "^4.0.1",
     "starwars-names": "^1.6.0",
     "stream-chat": "^8.25.1",
-    "stream-chat-react": "^11.13.0",
+    "stream-chat-react": "^11.19.0",
```

Comment:
It would be good if you upgrade the other sample apps as well.
`yarn upgrade-interactive` should help you with that. Also, we can bump `stream-chat` to the latest version.

Reply:
nice :)

Reply:
added